pkgs/development/libraries/science/math/osqp/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "osqp";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "oxfordcontrol";
+    repo = "osqp";
+    rev = version;
```

```suggestion
    rev = "v${version}";
```
This release is tagged at "v0.6.0". I didn't look closely enough. Doesn't build properly.